documentation/how-to/authorization.md

```diff
@@ -0,0 +1,246 @@
+# Authorization
+
+> **Note:** (TODO) Functionality described on this page is not live yet.
+
+Enabling authorization requires that you have an active
+[Charmed Temporal K8s Operator](https://discourse.charmhub.io/t/charmed-temporal-k8s-tutorial-introduction/11777)
+deployed as described in the tutorial. It is recommended that you first go
+through the steps of enabling authentication as outlined
+[here](./authentication.md).
+
+By default the Temporal Server doesn't offer any authorization, but it offers
+the plugin mechanism to add a custom one. We have added an OAuth-based
+[authentication](./authentication.md) using Google Cloud and an authorization
+mechanism that leverages [Google Cloud](https://cloud.google.com) and
+[OpenFGA](https://openfga.dev/). This custom build of the Temporal Server can be
+found [here](https://github.com/canonical/charmed-temporal-image).
+
+## Deploy OpenFGA
+
+To deploy Charmed OpenFGA K8s, you need to run the following command, which will
+fetch the charm from [Charmhub](https://charmhub.io/openfga-k8s) and deploy it
+to your model:
+
+```bash
+juju deploy openfga-k8s --channel edge
+```
+
+Wait until the application is ready - when it is ready, `juju status` will show:
+
+```
```

Comment:
Where is it written how to deploy all the other charms in the model? It might be nice to put a reference to how we got to his point.

Reply:
The first section of this page mentions this:

> Enabling authorization requires that you have an active [Charmed Temporal K8s Operator](https://discourse.charmhub.io/t/charmed-temporal-k8s-tutorial-introduction/11777) deployed as described in the tutorial.